src/dials/algorithms/integration/boost_python/integration_integrator_ext.cc

```diff
@@ -22,6 +22,9 @@ namespace dials { namespace algorithms { namespace boost_python {
    * Split the reflection table where the blocks are given.
    */
   inline void job_list_split(const JobList &self, af::reflection_table data) {
+    using dxtbx::af::flex_table_suite::reorder;
+    using dxtbx::af::flex_table_suite::setitem_column;
```

Comment:
Since there are now referring to namespaces defined outside of dials, would it be worth adding an explicit `#include <dxtbx/array_family/flex_table_suite.h>` instead of relying on the explicit one from `integrator.h`?